pyproject.toml

```diff
@@ -86,6 +86,12 @@ test = [
     "requests-unixsocket",
     "kfp-tekton"
 ]
+airflow = [
+
+]
+airflow-gitlab = [
```

Comment:
thank you very much, @lresende for adding this and the documentation.
I understand the install option airflow-gitlab, but what do the empty quare brackets in airflow do?
Just for my understanding.